src/node_config.cc

```diff
@@ -153,6 +153,13 @@ static void Initialize(Local<Object> target,
     v8EnvironmentFlags->Set(i, OneByteString(env->isolate(),
         v8_environment_flags[i]));
   }
+
+#if defined(NODE_REPORT)
+  const std::string& report_events = env->options()->report_events;
+  if (!report_events.empty()) {
+    READONLY_STRING_PROPERTY(target, "node_report", report_events);
```

Comment:
Use camel case for the property here. `s/node_report/nodeReport`

Reply:
subsequent reviews and changes thereon made this change un-necessary; thanks.